jenkins/README.md

```diff
@@ -98,12 +98,8 @@ sudo chmod 777 /workspace
 ### Deployment
 
 ```bash
-export JENKINS_IP=[...]
-
-# Replace MASTER_USER and MASTER_PASS with "real" values
-MASTER_USER=admin \
-    MASTER_PASS=admin \
-    docker stack deploy -c jenkins-swarm-agent.yml jenkins-agent
+    # Replace MASTER_USER and MASTER_PASS with "real" values
+    docker stack deploy -c jenkins-swarm-agent.yml jenkins-agent -username MASTER_USER -password MASTER_PASS
```

Comment:
Does it work by specifying `-username MASTER_USER -password MASTER_PASS` after the stack name? I'm probably missing something...

Reply:
Complete brain fart. Works only for `docker run` and `docker service create`.
`jenkins-swarm-agent.yml` already contains Envs expansion and current README.md is accurate.
So this PR is irrelevant, since this repo is about docker stacks.

Reply:
That was my mistake. I forgot that those are stacks. I'm sorry I wasted your time.